src/components/Restoration/BusinessName.vue

```diff
@@ -160,11 +160,10 @@ export default class BusinessName extends Mixins(CommonMixin, DateMixin, NameReq
    * @returns a promise to return the NR, or throws a printable error
    */
   async fetchAndValidateNr (nrNum: string, phone: string, email: string): Promise<NameRequestIF> {
-    const nameRequest = await LegalServices.fetchNameRequest(nrNum)
+    const nameRequest = await LegalServices.fetchValidContactNr(nrNum, phone, email)
     if (!nameRequest) throw new Error('Error fetching Name Request')
 
-    // validateNameRequest() already throws printable errors
-    return this.validateNameRequest(nameRequest, this.requestActionCode, phone, email)
```

Comment:
You should still call `validateNameRequest()` since it checks many other things.

Ultimately, we will be able to remove the email and phone check from validateNameRequest() but it doesn't hurt to leave it in there for now.

Reply:
Thanks for restoring this.